### PR TITLE
Add Show source on Github menu button

### LINKED ISSUE #90: Add "Contribute to this app" button in the Apple menu

### PR DESCRIPTION
**Pull request by wokalski**
Fixes #90 

image

### CODE REVIEW

File: renderer-macos/lib/bindings/BriskMenu.re [hunk 1]
```diff
@@ -1,7 +1,8 @@
 open CocoaTypes;
 
+type t = menu;
+
 type kind =
-  | Apple
```
Comment (rauanmayemir) [inline renderer-macos/lib/bindings/BriskMenu.re]: Why did you remove `Apple`?

File: renderer-macos/lib/bindings/BriskMenu.re [hunk 2]
```diff
@@ -45,66 +46,20 @@ type action =
   /* Custom callback */
   | Callback(unit => unit);
 
-type target =
```
Comment (rauanmayemir) [inline renderer-macos/lib/bindings/BriskMenu.re]: I just realized that we don't need anything but NSApp target as menu actions should be app-wide. Is that the case here?

File: examples/hacker-news/app/app.re [hunk 3]
```diff
@@ -1,15 +1,70 @@
...
   open Cocoa;
 
-  let appName = "BriskMac";
-
   Application.init();
 
   Application.willFinishLaunching(() => {
-    let menu = UiMenu.makeMainMenu(appName);
-    Menu.add(~kind=Main, menu);
+    open Brisk_macos.Menu;
```
Comment (rauanmayemir) [inline examples/hacker-news/app/app.re]: I badly want to keep entry point as thin as possible to make it easy for newbies to read it, but instead it's getting bloated.

File: renderer-macos/lib/bindings/BriskMenu.re (hunk 1 above)
Comment (wokalski) [reply renderer-macos/lib/bindings/BriskMenu.re]: Because apple is an unsupported legacy API.

File: renderer-macos/lib/bindings/BriskMenu.re (hunk 2 above)
Comment (wokalski) [reply renderer-macos/lib/bindings/BriskMenu.re]: All actions that are not `Callback` should have NSApp as the target. Specifiying it separately is a bit extra.

File: examples/hacker-news/app/app.re (hunk 3 above)
Comment (wokalski) [reply examples/hacker-news/app/app.re]: I'll put the menu under `MainMenu.menu` value.

File: renderer-macos/lib/bindings/BriskMenu.re (hunk 1 above)
Comment (rauanmayemir) [reply renderer-macos/lib/bindings/BriskMenu.re]: Gotcha. 🤦‍♂️
